### DISCUSSION TITLE
Make NX_DAEMON_SOCKET_DIR configurable via nx.json

### DISCUSSION THREAD

**Discussion by MarcusElevait** [nrwl/nx#23001] [Feature Requests]
We're currently consider using the NX_DAEMON_SOCKET_DIR variable to improve our local setup and use docker compose.
But this would mean every developer has to ensure by himself, that those env var is always set.
It would be cool, if we could set this option in the nx.json, so that everyone is automatically using it.

**Comment by jaysoo** [comment]
Could the environment variable by set in the `Dockerfile`? Is there a reason you'd want to change it for local development?

**Reply by MarcusElevait** [reply]
Maybe i have to clearify our setup. We've multiple nestjs backend services running in a docker container and the frontend is started on the local machine. And i want the frontend on the local machine and the backend in the docker containers to use the same docker socket. Therefore we need to have the env var set locally and in the docker container.
That being said, i think we can also move the frontend in a docker container, but not sure how the team likes this. So my initial request is still valid.

**Reply by jaysoo** [reply]
Developers could also set the environment variable in a `.env` file, and the repo could provide a `.env.sample` file as an example to copy from. Would that suffice as a workaround?

**Reply by MarcusElevait** [reply]
That would be sufficient, thank you
